package.json

```diff
@@ -25,7 +25,7 @@
     "archiver": "5.3.1",
     "async": "1.4.2",
     "chmodr": "1.0.2",
-    "door43-client": "^0.9.6",
+    "door43-client-fork": "file:src/js/lib/external/door43-client-fork-0.9.8.tgz",
```

Comment:
What do you think? Could we make a fork and create this package on npmjs.com, instead of loading this from file?

Reply:
thanks, that sounds great